modules/db/src/main/scala/gem/dao/SmartGcalDao.scala

```diff
@@ -0,0 +1,24 @@
+package gem.dao
+
+import gem.config.{F2SmartGcalKey, GcalConfig, SmartGcalKey}
+import gem.enum.{GcalBaselineType, GcalLampType}
+import doobie.imports._
+
+object SmartGcalDao {
+  def insert(l: GcalLampType, b: GcalBaselineType, k: SmartGcalKey, g: GcalConfig): ConnectionIO[Int] = {
+    def insertSmartF2(gcalId: Int, k: F2SmartGcalKey): ConnectionIO[Int] =
+      sql"""
+        INSERT INTO smart_f2 (lamp, baseline, disperser, filter, fpu, gcal_id)
+             VALUES ($l :: gcal_lamp_type, $b :: gcal_baseline_type, ${k.disperser}, ${k.filter}, ${k.fpu}, $gcalId)
```

Comment:
I think this is the only way to do it in a way that JDBC understands. The value is sent as a string.